packages/fxa-auth-server/lib/geocoding.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Client,
+  PlaceType2,
+  Status,
+} from '@googlemaps/google-maps-services-js';
+import countries from 'i18n-iso-countries';
+import { ConfigType } from '../config';
+import { Logger } from 'mozlog';
+const config: ConfigType = require('../config').getProperties();
+
+export type ZipLocation = {
+  city: string;
+  country: string;
+  countryCode: string;
+  state: string;
+  stateCode: string;
+};
+
+/**
+ *
+ * @param zip
+ * @param country - Two-letter country code ([ISO 3166-1 alpha-2] or corresponding Country name
+ * @returns
+ */
+export async function getLocationFromZip(
+  log: Logger,
+  zip: string,
+  country: string = 'US'
+): Promise<ZipLocation | {}> {
+  const client = new Client();
+  const countryName =
+    country.length === 2 ? countries.getName(country, 'en') : country;
```

Comment:
The Geocoding API needs the full Country Name, and it seems that at least on the `Stripe.Card` the `country` is a 2 letter ISO code.

I added this logic, just in case we pass in the Country Name. Removing it shouldn't impact anything else.

Reply:
Hmm yeah if we don't have a use case currently to accept more than the two-letter country code, I wouldn't add support for more than that just to keep things simple. Are there `CountryCode` types or something like that from one of these new libraries we're using in this new module? That'd be the best way to ensure we aren't passed something we don't expect...